app/src/app/pages/genre/genre.component.html

```diff
@@ -3,17 +3,11 @@
     <div class="col-sm-7" *ngIf="genre">
       <app-title [entity]="genre"></app-title>
 
-      <div class="mb-3 mt-5"></div>
-
-      <div *ngIf="genre?.abstract">
-        <app-collapse>
-          <app-abstract [entity]="genre"></app-abstract>
-        </app-collapse>
-      </div>
+      <app-information label="Links" [sourceID]="genre?.sourceID"></app-information>
```

Comment:
Please add i18n translation to this, even though it should be the same in German.